src/swagger-specs/flow-service.yaml

```diff
@@ -35,26 +35,27 @@ host: platform.adobe.io
   - name: Flows
-    description: Flows represent the data transfer between a source connection and a target connection. This also includes information regarding the ingestion schedule and the transformation specifications of the data being transferred from source to target. A flow instance is specific to a tenant and IMS Organization.
+    description: Flows (or dataflows) represent the transfer of data between source and target. They include details on the transformations applied to your data, the current status of your flow, and details on the most recent run iteration of your flow. Flows also contain metadata, attributes inherited from the base, source, and target connections, as well as a unique flow ID. An instance of a flow is specific to an organization.
   - name: Flow specs 
-    description: Flow specifications contain the information and details that define a flow. Flow specifications are tenant and IMS Organization agnostic.
+    description: Flow specs retain information that define a flow. They include information on how a flow can be scheduled, whether a flow supports options like partial ingestion and error diagnostics, as well as the parameters required for different mappings from source to target. Flow specs also include information on how a flow run can be created and a list of connection spec IDs and target connection IDs that a given flow can support. Flow specs are organization agnostic.
   - name: Runs
-    description: Runs represent an instance of flow execution. A run instance is specific to a tenant and IMS Organization.
+    description: Runs (or flow runs) represent the instance of a flow execution. For example, if a flow is scheduled to run hourly at 9 AM, 10 AM, and 11 AM, then you would have three instances of a run. Runs retain metric summaries on your flow executions, including information on the duration of the run, size of the data, file count, and a status update. An instance of a run is specific to an organization.
   - name: Source connections
-    description: Source connections create and manage a connection to the external source from where data is ingested. Source connections contain information like data source, data format, and the source connection ID needed to create a flow. A source connection instance is specific to a tenant and IMS Organization.
+    description: Source connections represent the incoming data from a given source. Source connections define where the data is coming from and in what format. Source connections also refer to the base connection for authentication information. For file-based sources like cloud storages, source connections can define additional information on the data transfer, such as column delimiter, encoding type, compression type, regular expressions, and whether ingestion is to be done recursively. An instance of a source connection is specific to an organization.
```

Comment:
Can we rephrase it to: "Source connections define the origin and format of the data."